### PR TITLE
[Form] Update form.rst for flattened form errors

### PR DESCRIPTION
**Pull request by DocFX**
Updated to reflect the code.

### CODE REVIEW

File: components/form.rst [hunk 1]
```diff
@@ -750,7 +750,7 @@ method to access the list of errors. It returns a
     $errors = $form['firstName']->getErrors();
 
     // a FormErrorIterator instance in a flattened structure
-    $errors = $form->getErrors(true);
+    $errors = $form->getErrors(false, true);
```
Comment (xabbuh) [inline components/form.rst]: I think the previous example is correct. We do want to get the errors from nested forms (that's why the first argument should be `true`), but we want to flatten the structure (the second argument defaults to `true`).

Comment (DocFX) [reply components/form.rst]: I think I got somehow confused by this. I actually had to read the docs many times to get it, I got better help by reading the method signature in the end.
Maybe the comment is wrong, then?

Shouldn't this be sth like:
```php
    // a FormErrorIterator instance including child forms in a flattened structure
    $errors = $form->getErrors(true);
    
    // a FormErrorIterator instance including child forms without flattening the output structure
    $errors = $form->getErrors(true, false);
```

Comment (xabbuh) [reply components/form.rst]: we could indeed update the comment to what you suggest to clarify what is happening here

Comment (DocFX) [reply components/form.rst]: Done and updated. :)

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you William.

**Pull request merged**